session/session.go

```diff
@@ -3048,3 +3058,7 @@ func (s *session) updateTelemetryMetric(es *executor.ExecStmt) {
 func (s *session) GetBuiltinFunctionUsage() map[string]uint32 {
 	return s.builtinFunctionUsage
 }
+
+func (s *session) getSnapshotInterceptor() kv.SnapshotInterceptor {
+	return temptable.SessionSnapshotInterceptor(s)
```

Comment:
Can you add a test case to verify that the snapshot interceptor exists?
For example, begin a transaction, do some batch get, and get the snapshot interceptor through the session.